jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/SystemConfigMBean.java

```diff
@@ -0,0 +1,17 @@
+package us.dot.its.jpo.ode;

```

Comment:
I don't see the point of this interface. Looks like it was written for a tutorial purpose and NOT relevant to our applications.